sdk/src/contracts/collections/features/grantRole.ts

```diff
@@ -0,0 +1,91 @@
+import { BigNumber, BytesLike, CallOverrides, ContractTransaction, ethers, PopulatedTransaction } from 'ethers';
+import { AccessControl, Addressish, asAddress, CollectionContract, Signerish } from '../..';
+import { SdkError, SdkErrorCode } from '../errors';
+import { asCallableClass, CatchAllInterfaces, ContractFunction } from './features';
+
+const GrantRoleFunctions = {} as const;
+
+const GrantRolePartitions = {
+  // 'GrantRole' is always present but not actually exposed by any interface
+  catchAll: CatchAllInterfaces,
+};
+type GrantRolePartitions = typeof GrantRolePartitions;
+
+const GrantRoleInterfaces = Object.values(GrantRolePartitions).flat();
+type GrantRoleInterfaces = (typeof GrantRoleInterfaces)[number];
+
+export type GrantRoleCallArgs = [signer: Signerish, role: BytesLike, account: Addressish, overrides?: CallOverrides];
```

Comment:
For write functions we have `WriteOverrides`:
```suggestion
export type GrantRoleCallArgs = [signer: Signerish, role: BytesLike, account: Addressish, overrides?: WriteOverrides];
```